Framework/PythonInterface/plugins/algorithms/AlignAndFocusPowderFromFiles.py

```diff
@@ -621,6 +621,12 @@ def __setupCalibration(self, wksp):
         if loadCalibration or loadGrouping or loadMask:
             if not wksp:
                 raise RuntimeError("Trying to load calibration without a donor workspace")
+
+            if "TMin" in self.__getAlignAndFocusArgs():
```

Comment:
Since `self.__getAlignAndFocusArgs()` returns a `dict`, this could be simplified to
```py
tmin_val = self.__getAlignAndFocusArgs().get("TMin", 0.0)
```
which does the same thing